test/forms/capitalallowances/balancingAllowance/BalancingAllowanceAmountFormProviderSpec.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.capitalallowances.balancingAllowance
+
+import forms.behaviours.BigDecimalFieldBehaviours
+import models.common.UserType.Individual
+import models.common.{MoneyBounds, UserType}
+import org.scalacheck.Gen
+import play.api.data.{Form, FormError}
+
+class BalancingAllowanceAmountFormProviderSpec extends BigDecimalFieldBehaviours with MoneyBounds {
+
```

Comment:
Can replace all of this with the CurrencyFormProviderBaseSpec extension like the other amount pages if you like